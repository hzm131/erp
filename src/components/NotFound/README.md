see https://preview.pro.ant.design/#/exception/404
