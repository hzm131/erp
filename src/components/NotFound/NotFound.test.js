import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './NotFound'

it('should render using shallow', ()=>{
  const shallowed = shallow(<NotFound />)

  expect(shallowed).toMatchSnapshot()
})
