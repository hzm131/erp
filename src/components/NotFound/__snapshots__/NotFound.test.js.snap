// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render using shallow 1`] = `
<Exception
  type="404"
/>
`;
